Dockerfile

```diff
@@ -83,7 +83,8 @@ RUN version="7.3.6" && \
     find "${install_dir}" -type f -exec chmod -x "{}" ";" && \
     chmod +x "${install_dir}/pwsh" && \
     ln -s "${install_dir}/pwsh" /usr/bin/pwsh && \
-    pwsh --version
+    pwsh --version && \
+    rm -rf /tmp/.dotnet
```

Comment:
`pwsh --version` creates and uses some tmp directories
(where it seemingly didn't before),  and leaves an empty `/tmp/.dotnet/shm` directory. Although that's not an issue per se, forcing cleanup here because we check /tmp is empty in setup.py https://github.com/ansible/base-test-container/blob/main/files/setup.py#L18.

Maybe related to https://github.com/dotnet/runtime/commit/ae1bff097cae9604c52f90302a665349ca293918 or https://github.com/dotnet/runtime/commit/1c4e4c11e4ad514a6a1198ce494873365af4eace?

strace of `pwsh --version` with 7.3.6 to 7.4.0-preview.5: https://gist.github.com/s-hertel/baeebdf831da16d1b70b10ff7c66e23a#file-gistfile1-txt-L3046

strace with 7.3.7 https://gist.github.com/s-hertel/eccd28e51693ea866d853a870a70decb